### PR TITLE
feat: implement OR opcode

### LINKED ISSUE #22: Opcode: OR

### PR DESCRIPTION
**Pull request by Danikim01**
closes #22 
This PR implements the OR opcode

### CODE REVIEW

File: tests/operations.rs [hunk 1]
```diff
@@ -269,3 +269,22 @@ fn jumpdest() {
...
+fn test_or() {
+    let a = BigUint::from(0b1010_u8);
+    let b = BigUint::from(0b1110_u8);
+    let expected = BigUint::from(0b1110_u8);
+    let program = vec![
+        Operation::Push(a.clone()),
+        Operation::Push(b.clone()),
+        Operation::Or,
+    ];
+    run_program_assert_result(program, expected.try_into().unwrap());
```
Comment (Oppen) [inline tests/operations.rs:283]:
```suggestion
    let expected = 0b1110_u8;
    let program = vec![
        Operation::Push(a.clone()),
        Operation::Push(b.clone()),
        Operation::Or,
    ];
    run_program_assert_result(program, expected);
```

File: README.md [hunk 2]
```diff
@@ -19,9 +19,11 @@ An EVM-bytecode to machine-bytecode compiler using MLIR and LLVM.
 1. (0x04) DIV
 1. (0x06) MOD
 1. (0x08) ADDMOD
+
```
Comment (Oppen) [inline README.md:22]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -19,9 +19,11 @@ An EVM-bytecode to machine-bytecode compiler using MLIR and LLVM.
...
 1. (0x13) SGT
 1. (0x18) XOR
 1. (0x1A) BYTE
+1. (0x17) OR
```
Comment (Oppen) [inline README.md:26]: Please keep ordered.

File: README.md [hunk 4]
```diff
@@ -70,13 +72,12 @@ An EVM-bytecode to machine-bytecode compiler using MLIR and LLVM.
 1. (0x09) MULMOD
 1. (0x0A) EXP
 1. (0x0B) SIGNEXTEND
-1. (0x10) LT
```
Comment (Oppen) [inline README.md:73]: This shouldn't be removed.

**Review by Oppen — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Danikim01) [reply README.md]: resolved

**Review by Oppen — APPROVED** [review]
Just need to fix conflicts.

**Pull request merged**